drivers/system_pathways/app/models/system_pathways/chart_base.rb

```diff
@@ -141,13 +144,84 @@ def average(value, count)
       scope.merge(race_scope)
     end
 
+    private def filter_for_race_with_details_filter(scope, race_filter)
+      return scope.where(id: multi_racial_clients(scope).select(:id)) if race_filter.races.include?('MultiRacial')
+
+      query = races.except('MultiRacial', 'RaceNone').keys.map { |k| [k.underscore.to_sym, false] }.to_h
+      race_filter.races.each do |column|
+        next if column.in?(['MultiRacial', 'RaceNone'])
+
+        query[column.underscore.to_sym] = true
+      end
+      scope.where(query)
```

Comment:
Does this need:
```suggestion
      scope.where(**query)
```

Reply:
It doesn't seem to matter as the signature of `where` is `*args`:
```ruby
def foo(*args)
  puts args.inspect
end

foo(a: 1, b: 2)
[{:a=>1, :b=>2}]

foo({a: 1, b: 2})
[{:a=>1, :b=>2}]

```
But, if it seems more readable, I'm happy to change it.

As to the race columns, they are [true, false] by construction:
```ruby
am_ind_ak_native: client.am_ind_ak_native == 1,
asian: client.asian == 1,
...
```
But, the DB does mark them as `nullable`, so we could clean it up if desired. I believe there are other places where there are non-null assumptions that we should look at if this should be changed.